docs/en/install-upgrade/upgrading-stack.asciidoc

```diff
@@ -52,15 +52,42 @@ endif::[]
 compatible with {es} version {version}.
 
 . Test the upgrade in an isolated environment before upgrading your production
-cluster.
+  cluster.
++
+[IMPORTANT]
+====
+The upgraded version of {es} may interact with its environment in different
+ways from the version you are currently running. It is possible that your
+environment behaves incorrectly in a way that does not matter to the version of
+{es} that you are currently running, but which does matter to the upgraded
+version. In this case, the upgraded version will not work correctly until you
+address the incorrect behaviour in your environment.
+
+During your upgrade tests, pay particular attention to the following aspects:
+
+Cluster stability:: Does the new version of {es} form a stable healthy cluster?
+
+Indexing and search performance:: Does the new version of {es} perform the same
+(or better) than the current one on your specific workload and data?
+
+Snapshots:: Do all of your snapshot repositories work correctly and pass
+{ref}/repo-analysis-api.html[repository analysis]?
```

Comment:
```suggestion
* *Cluster stability*: Does the new version of {es} form a stable healthy cluster?
* *Indexing and search performance*: Does the new version of {es} perform the same
(or better) than the current one on your specific workload and data?
* *Snapshots*: Do all of your snapshot repositories work correctly and pass
{ref}/repo-analysis-api.html[repository analysis]?
```

Reply:
this might be a workaround to the squishiness

Reply:
alternatively, just add a `+` to the blank line before "Snapshots" and see if that breaks it up?????